source/plugins/ruby/out_mdm.rb

```diff
@@ -118,12 +118,10 @@ def start
               @useMsi = true
               msi_endpoint = @@imds_msi_endpoint_template % { resource: @@token_resource_audience }
               @parsed_token_uri = URI.parse(msi_endpoint)
-              @cached_access_token = get_access_token
```

Comment:
So get_access_token will still retry for a total of 6 minutes when the AAD pod identity is enabled without the exception, but this is ok since now it will only block the MDM metrics, which wouldn't be able to authenticate to send anyway?

Reply:
> So get_access_token will still retry for a total of 6 minutes when the AAD pod identity is enabled without the exception, but this is ok since now it will only block the MDM metrics, which wouldn't be able to authenticate to send anyway?

>  This is not issue in case of the MSI auth mode (meaning with DCRs) for customers with AAD Pod identity in standard mode. But, the agent which is onboarded without MSI/DCR based auth, IMDS call failed if the customer (with AAD Pod identity in standard mode) not added the exception and this **only** impact the MDM metrics.